### DISCUSSION TITLE
can we bypass region block on pc

### DISCUSSION THREAD

**Discussion by m1s3rys1gn4l** [redphx/better-xcloud#535] [General]
hello,
xbox cloud is not supported in my region, have tried xcloud in android there is a option called region bypass, it was working great.
now i would like to inform there is no option in pc for bypass region,
am i missing something ?

**Comment by redphx** [comment]
It should be there. Screenshot?

**Reply by m1s3rys1gn4l** [reply]
![Screenshot 2024-10-23 123903](https://github.com/user-attachments/assets/f7e83248-06bc-4dd7-9e9b-ad91cf8e46c1)
here is screenshot

**Comment by m1s3rys1gn4l** [comment]
its solved by enabled developer mode in extension,
thanks :)
